stl/inc/xstring

```diff
@@ -169,11 +179,19 @@ struct _WChar_traits { // char_traits for the char16_t-likes: char16_t, wchar_t,
 #endif // _HAS_CXX17
     }
 
-    static _Elem* copy(_Out_writes_all_(_Count) _Elem* const _First1, _In_reads_(_Count) const _Elem* const _First2,
-        const size_t _Count) noexcept /* strengthened */ {
+    static _CONSTEXPR20_MEMOP _Elem* copy(_Out_writes_all_(_Count) _Elem* const _First1,
+        _In_reads_(_Count) const _Elem* const _First2, const size_t _Count) noexcept /* strengthened */ {
         // copy [_First2, _First2 + _Count) to [_First1, ...)
+#ifdef __clang__
+        if constexpr (is_same_v<_Elem, wchar_t>) {
+            __builtin_wmemcpy(_First1, _First2, _Count);

```

Comment:
Are we missing a return statement here?

Reply:
I just pushed a fix. Thanks for finding this!